project_hr/models/project_task.py

```diff
@@ -0,0 +1,95 @@
+# Copyright 2018 Tecnativa - Pedro M. Baeza
+# Copyright 2019 Brainbean Apps (https://brainbeanapps.com)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import _, api, exceptions, fields, models
+
+
+class ProjectTask(models.Model):
+    _inherit = "project.task"
+
+    employee_id = fields.Many2one(
```

Comment:
This field should be changed as well as the `user_ids` to many2many `employee_ids`.

Reply:
Hello @pedrobaeza , Thank you for your review.
I have added the requested changes. You can have a look.